### PR TITLE
feat: update kotlin version

### PR DESCRIPTION
**Pull request by dawnfire05**
## Summary by CodeRabbit

- **버그 수정**
	- Kotlin 플러그인 선언이 업데이트되어 호환성과 기능이 향상되었습니다.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ plugins {
     id 'com.google.gms.google-services'
     id 'com.google.firebase.crashlytics'
     // END: FlutterFire Configuration
-    id "kotlin-android"
+    kotlin("jvm") version "2.0.21"
```
[the remark this answers is not in the document]
Comment (dawnfire05) [reply android/app/build.gradle]: 똑똑한 것 보소

**Comment by dawnfire05** [comment]
@2paperstar #424 android buuild test 실패 원인 해결입니당

**Comment by 2paperstar** [comment]
요거는 한번 따로 체크해보고 리뷰 남길게요

**Comment by dawnfire05** [comment]
> 요거는 한번 따로 체크해보고 리뷰 남길게요

넵

**Comment by 2paperstar** [comment]
@dawnfire05 근데 #424 에서 android 관련해서 건든게 없는데, 터지면 안 되는거 아닌가요? 혹시 파악된 원인이 있을까요?

**Comment by dawnfire05** [comment]
@2paperstar
`e: /home/runner/.gradle/caches/transforms-3/911b38c980c3e15b0e3f68c26e582cc8/transformed/jetified-kotlin-stdlib-1.9.24.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.9.0, expected version is 1.7.1.
e: /home/runner/.gradle/caches/transforms-3/911b38c980c3e15b0e3f68c26e582cc8/transformed/jetified-kotlin-stdlib-1.9.24.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.9.0, expected version is 1.7.1.
e: /home/runner/.gradle/caches/transforms-3/911b38c980c3e15b0e3f68c26e582cc8/transformed/jetified-kotlin-stdlib-1.9.24.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.9.0, expected version is 1.7.1.
e: /home/runner/.gradle/caches/transforms-3/fcae050553e3810b228290f4142742c4/transformed/jetified-annotation-jvm-1.9.1.jar!/META-INF/annotation.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.9.0, expected version is 1.7.1.
`
https://stackoverflow.com/questions/78422586/error-module-was-compiled-with-an-incompatible-version-of-kotlin-the-binary-ve

이게 에러로 추정되는 원인인데
뭐하면서 에러 생긴지는 모르겠어요,,

**Comment by 2paperstar** [comment]
#424 액션 잘 돌아가네요! 다른 deps쪽에 문제가 있는 것 같습니다
아마 flutter 3.27이 추가 되면 build.gradle 쪽에 변화가 있을 것 같은데, 그거랑 같이 핸들링하면 될 것 같아요

**Comment by 2paperstar** [comment]
이거는 close 하시죠

**Pull request closed without merging**